secretariat/models.py

```diff
@@ -48,3 +50,19 @@ class Membership(models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     organisation = models.ForeignKey(Organisation, on_delete=models.CASCADE)
     role = models.CharField(max_length=15)
+    start_date = models.DateField(null=True, blank=True)
+    end_date = models.DateField(null=True, blank=True)
+
+    class Meta:
+        constraints = [
+            CheckConstraint(
+                check=Q(end_date__gt=F("start_date")),
```

Comment:
Je pense qu'il faut aussi gérer les cas où l'une ou l'autre date est vide : 

```suggestion
                check=Q(end_date__gt=F("start_date"))|Q(end_date__isnull=True)|Q(start_date__isnull=True),
```

(syntaxe pas testée du tout, j'y vais à l'audace)

Reply:
oui, c'était prévu ! en fait j'aurais dû repasser cette PR en draft hier soir